src/components/GridFeatured/GridFeatured.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+    <div class="grid-featured">
+        <block-featured
+            v-for="(block, i) in blocks"
+            :key="i"
```

Comment:
Don't use index as key! Use `block.id`. Yes this will through errors in Storybook, that is OK.

![1bb1952285f30288900a434ddce22e52](https://user-images.githubusercontent.com/475015/86998763-ea996300-c165-11ea-88c7-5dd1b1c03394.jpg)